Mailer/app/models/Repositories/LogsRepository.php

```diff
@@ -28,6 +28,13 @@ class LogsRepository extends Repository
         'suppress-unsubscribe' => 'hard_bounced_at',
     ];
 
+    public function all()
+    {
+        // TODO column sorting does not exist
+//        return $this->getTable()->order('sorting ASC');
+        return $this->getTable();
```

Comment:
You can use `created_at DESC` as a default sort. With emails, it's always like this.

Reply:
My bad, please don't. Should be explicitly handled by the handlers/presenters.